### PR TITLE
Alow http1.1 pipelining

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: src/libsync/httplogger.cpp [hunk 1]
```diff
@@ -47,10 +47,14 @@ void logHttp(const QByteArray &verb, const QString &url, const QByteArray &id, c
...
-        stream << " " << reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt();
+        stream << " " << reply->attribute(QNetworkRequest::HttpStatusCodeAttribute).toInt() << " (";
         if (reply->error() != QNetworkReply::NoError) {
-            stream << " (Error: " << reply->errorString() << ")";
+            stream << "Error: " << reply->errorString() << ",";
         }
+        if (reply->attribute(QNetworkRequest::HttpPipeliningWasUsedAttribute).toBool()) {
+            stream << "Piplined,";
+        }
+        stream << ")";
```
Comment (TheOneRing) [inline src/libsync/httplogger.cpp:57]:
```
07-20 11:12:33:616 [ info sync.httplogger ]:	"931966b6-4f59-4099-987d-886b53b51110: Response: GET 200 (Piplined,) https://demo.owncloud.com/ocs/v2.php/apps/notifications/api/v1/notifications?format=json Header: { Cache-Control: no-cache, no-store, must-revalidate, Content-Length: 108, Content-Security-Policy: default-src ...
```
Could be a bit more pretty.

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/12534/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/12534/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/12534/guiReportUpload/serverlog.log

Screenshots:
- [sharee_edits_content_of_files_shared_by_sharer.png](https://cache.owncloud.com/public/owncloud/client/12534/guiReportUpload/screenshots/sharee_edits_content_of_files_shared_by_sharer.png)

</details>

File: changelog/unreleased/9930 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+Enhancement: Allow http1.1 pipelining
```
Comment (fmoc) [inline changelog/unreleased/9930:1]:
```suggestion
Enhancement: Allow HTTP/1.1 pipelining
```

File: changelog/unreleased/9930 [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+Under certain conditions that can result in a better network utilisation.
```
Comment (fmoc) [inline changelog/unreleased/9930:3]:
```suggestion
Under certain conditions, this change can result in a better network utilization.
```

**Review by fmoc — CHANGES_REQUESTED** [review]

**Review by fmoc — APPROVED** [review]

**Pull request merged**